src/main/java/org/catmaid/ScaleCATMAID.java

```diff
@@ -72,46 +72,55 @@
 	{
 		public int tileWidth;
 		public int tileHeight;
+		public long minC;
+		public long maxC;
+		public long minR;
+		public long maxR;
 		public long minZ;
 		public long maxZ;
 		public String tileFormat;
 		public String format;
 		public float quality;
 		public int type;
+		public boolean ignoreEmptyTiles;
 	}
-	
-	private ScaleCATMAID(){}
-	
-	static protected Param parseParameters()
-	{
+
+	private ScaleCATMAID() {}
+
+	static protected Param parseParameters() {
 		final Param p = new Param();
 		p.tileWidth = Integer.parseInt( System.getProperty( "tileWidth", "256" ) );
 		p.tileHeight = Integer.parseInt( System.getProperty( "tileHeight", "256" ) );
-		
+
+		p.minC = Long.parseLong( System.getProperty( "minC", "0" ) );
+		p.maxC = Long.parseLong( System.getProperty( "maxC", "" + Integer.MAX_VALUE ) );
+		p.minR = Long.parseLong( System.getProperty( "minR", "0" ) );
+		p.maxR = Long.parseLong( System.getProperty( "maxR", "" + Integer.MAX_VALUE ) );
 		p.minZ = Long.parseLong( System.getProperty( "minZ", "0" ) );
 		p.maxZ = Long.parseLong( System.getProperty( "maxZ", "" + Long.MAX_VALUE ) );
 		final String basePath = System.getProperty( "basePath", "" );
-		p.tileFormat = System.getProperty( "tileFormat", basePath + "%5$d/%8$d_%9$d_%1$d.jpg" );
-		
-		System.out.println( p.tileFormat );
-		
 		p.format = System.getProperty( "format", "jpg" );
+		final String tileFormat = System.getProperty( "tileFormat", "%5$d/%8$d_%9$d_%1$d");
+		p.tileFormat = basePath + "/" + tileFormat + "." + p.format;
```

Comment:
Definitely no.  Perlman wanted to use other streaming mechanisms that may not end in "." + p.format.  I would rather get rid of basePath although it has been useful for lazy parameter setting.